client/GameBoard.jsx

```diff
@@ -286,49 +286,80 @@ export class InnerGameBoard extends React.Component {
+                                    <CardCollection className={thisPlayer.plotSelected ? 'plot plot-selected' : 'plot'}
+                                                    title='Plots' source='plot deck' cards={thisPlayer.plotDeck} topCard={{ facedown: true, kneeled: true }} orientation='horizontal'
+                                                    onMouseOver={this.onMouseOver} onMouseOut={this.onMouseOut} onCardClick={this.onCardClick} onDragDrop={this.onDragDrop} />
+                                </div>
+                            </div>
+                            <div className='middle-right'>
+                                <div className='inset-pane'>
+                                    <div className={'phase-indicator ' + thisPlayer.phase}>{thisPlayer.phase} phase</div>
+                                    <MenuPane title={thisPlayer.menuTitle} buttons={thisPlayer.buttons} onButtonClick={this.onCommand}
+                                                onMouseOver={this.onMouseOver} onMouseOut={this.onMouseOut} />
+                                </div>
+                                <div className='schemes-pane'>
                                 </div>
                             </div>
-
-                            <PlayerStats gold={thisPlayer.gold || 0} claim={thisPlayer.claim || 0} reserve={thisPlayer.reserve || 0}
-                                power={thisPlayer.totalPower} isMe={!this.state.spectating} user={thisPlayer.user} />
                         </div>
-                        <div className='inset-pane'>
-                            <div />
-                            <MenuPane title={thisPlayer.menuTitle} buttons={thisPlayer.buttons} onButtonClick={this.onCommand}
-                                        onMouseOver={this.onMouseOver} onMouseOut={this.onMouseOut} />
+                        <div className='player-info our-side'>
+                            <PlayerStats gold={thisPlayer.gold || 0} claim={thisPlayer.claim || 0} reserve={thisPlayer.reserve || 0}
+                                        power={thisPlayer.totalPower} isMe={!this.state.spectating} user={thisPlayer.user} />
+                            <div className='deck-info'>
+                                <div className={'first-player-indicator ' + (thisPlayer.firstPlayer ? '' : 'hidden')}>First player</div>
+                                <div className='deck-type'>
+                                    <CardCollection className='faction' source='faction' cards={[]} topCard={thisPlayer.faction} onMouseOver={this.props.onMouseOver} onMouseOut={this.props.onMouseOut} disablePopup />
+                                    {thisPlayer.agenda && thisPlayer.agenda.code !== '' ?
+                                        <CardCollection className='agenda' source='agenda' cards={[]} topCard={thisPlayer.agenda} onMouseOver={this.props.onMouseOver} onMouseOut={this.props.onMouseOut}
```

Comment:
It looks like menus on Agenda cards are broken now (try testing with a Fealty agenda deck). This is because `this.props.onMenuItemClick` does not exist. However, `this.onMenuItemClick` does.